coordinator/src/routes.rs

```diff
@@ -604,3 +617,132 @@ pub async fn version() -> Result<Json<Version>, AppError> {
         branch: env!("BRANCH_NAME").to_string(),
     }))
 }
+
+pub async fn collaborative_revert_confirm(
```

Comment:
🔧 Moving the actual logic into a separate function would simplify the error handling and improve maintainability. IMHO we shouldn't have much logic in the API layer.

Reply:
[`a6e2598` (#1478)](https://github.com/get10101/10101/pull/1478/commits/a6e259816dbd3f35f27ef04a904f6f3ecdbb4eed)